### ISSUE TITLE
win_shell module not working on Windows Server 2016

### ISSUE THREAD

**Issue by mattdetine** [ansible/ansible#50042]
##### SUMMARY

When using the win_shell module on a Windows Server 2016 host, I receive the following error:

fatal: [bamq101bvwa]: FAILED! => {"changed": false, "cmd": "echo \"Hello world!\"", "msg": "Exception calling \"RunCommand\" with \"5\" argument(s): \"Failed to create new process (Access is denied, Win32ErrorCode 5)\"", "rc": 2}

Using a pre 2.7 version of Ansible or a Windows Server 2008 or 2012R2 resolves this issue.
  
##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

win_shell
##### ANSIBLE VERSION

```paste below
ansible 2.7.4
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/y15061/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, May  3 2017, 07:55:04) [GCC 4.8.5 20150623 (Red Hat 4.8.5-14)]

```

##### CONFIGURATION

```paste below
HOST_KEY_CHECKING(/etc/ansible/ansible.cfg) = False
```

##### OS / ENVIRONMENT

Microsoft Windows Server 2016 Standard
##### STEPS TO REPRODUCE

```yaml
---
- name: Hello World!
  hosts: all
 
  tasks:
 
  - name: Hello World!
    win_shell: echo "Hello world!"
```

##### EXPECTED RESULTS

Standard out return of Hello world!

##### ACTUAL RESULTS

```paste below
fatal: [bamq101bvwa]: FAILED! => {"changed": false, "cmd": "echo \"Hello world!\"", "msg": "Exception calling \"RunCommand\" with \"5\" argument(s): \"Failed to create new process (Access is denied, Win32ErrorCode 5)\"", "rc": 2}
```

**Comment by ansibot** [comment]
Files identified in the description:
* [lib/ansible/modules/windows/win_shell.ps1](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_shell.ps1)

If these files are inaccurate, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @dagwieers @jborean93 @jhawkesworth @nitzmahone
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by jborean93** [comment]
The only change made to `win_shell` in devel/2.7 is https://github.com/ansible/ansible/pull/47404 but technically the code shouldn't be running for the base `winrm` connection plugin as a console already exists. If the user you are running with an admin user or a standard user on the 2016 host?

Are you able to try out the following;

* Use `win_command: powershell.exe echo "Hello world!"` as the task
* Install 2.7.0 with `pip install ansible==2.7.0` as the PR I linked above was only backported for 2.7.1 and above
* Try out devel, https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#running-from-source, as that has a few more changes incorporated that may help

**Comment by mattdetine** [comment]
I believe we have identified our issue here. We use CrowdStrike Falcon as an endpoint protection software. Out of the blue it started killing off any new process spawned using the win_shell module. I have tested the win_command module which does not set off CrowdStrike for some reason. We're working with our security team to see if we can get our ansible IPs whitelisted. I apologize for the erroneous bug report. Thanks for the quick response!
